packages/mobile/src/components/manager-mode-drawer/useShowManagerModeNotAvailable.tsx

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react'
+
+import { useDrawer } from 'app/hooks/useDrawer'
+
+import { useRoute } from 'app/hooks/useRoute'
+
+export const useShowManagerModeNotAvailable = () => {
+  const { params } = useRoute()
+  const { onOpen: openManagerModeDrawer } = useDrawer('ManagerMode')
+  useEffect(() => {
+    if ((params as { path: string })?.path === 'accounts-you-manage') {
+      openManagerModeDrawer()
```

Comment:
Do we need to do any modification of the path in this case so that navigating back from another screen doesn't trigger the drawer again?

Reply:
good q - we do not because the path won't stick around when you navigate back